src/EnergyPlus/RuntimeLanguageProcessor.cc

```diff
@@ -2438,6 +2456,90 @@ namespace RuntimeLanguageProcessor {
                                                                                             // independent | 5th independent
                         }
 
+                    } else if (SELECT_CASE_var == FuncTodayIsRain) {
+                        TodayTomorrowWeather(
+                            FuncTodayIsRain, Operand(1).Number, Operand(2).Number, WeatherManager::TodayIsRain, ReturnValue);
```

Comment:
New functions all call the same common `TodayTomorrowWeather` function, passing in the appropriate data array to access.